docs/get_started/plan_deployment/traditional_deployment/plan_multiple_profile.md

```diff
@@ -1,47 +1,47 @@
-4.  Run the enable-profiles or replace-profiles configuration task to capture the Portal configuration in the profile template.
-5.  Import the saved search collections on the original profile.
-6.  Create new profiles with the profile templates; default search collections are automatically created in the new profile.
-7.  Run the chmod -R g+rx /portal\_server\_root task to restore permissions to the Portal Server directory.
+1.  Export the search collections.
+2.  Remove the search collections.
+3.  If you installed HCL Portal as a non-root user, run the `chmod -R g+rwx /portal\_server\_root` task to modify permissions for the Portal Server directory.
+4.  Run the enable-profiles or replace-profiles configuration task to capture the portal configuration in the profile template.
+5.  Import the saved search collections from the original profile.
+6.  Create profiles with the profile templates. The default search collections are automatically created in the new profile.
+7.  Run the `chmod -R g+rx /portal\_server\_root` task to restore permissions to the Portal Server directory.
 
-If you want to share search collections between multiple server instances, then you must configure a remote search server that supports search collections sharing.
+To share search collections between multiple server instances, you must configure a remote search server that supports sharing search collections.
 
-## Important information for cluster configurations
+## Cluster configuration requirements and maintenance
+<!-- Presumably, all the information we offer has some level of importance, especially info in sections with a subhead. We can highlight short bits of highly important info with an "Important" note.  -->
+Manage all maintenance at the cluster level. During maintenance, take the entire cluster out of service. By working at this level, the cluster-wide changes take effect without affecting user traffic or potentially causing synchronization conflicts. In a continuous availability environment, multiple clusters might be required to allow traffic to continue to one cluster while another is being serviced.
 
-Manage all maintenance at the cluster level. During maintenance, take the entire cluster out of service. Then, the cluster-wide changes can take effect without affecting user traffic or potentially causing synchronization conflicts. In a continuous availability environment, multiple clusters might be necessary to allow traffic to continue to one cluster while another is being serviced.
+Maintaining an HCL Portal installation with multiple profiles involves  maintaining the product binary files and maintaining each profile instance. All profile instances and the HCL Portal product binary files are updated at the same time to avoid conflicts. If a profile belongs to a cluster and the cluster is updated, the shared HCL Portal product binary files and all other profiles must be updated to maintain synchronization.
```

Comment:
```suggestion
Maintaining an HCL Portal installation with multiple profiles involves maintaining the product binary files and maintaining each profile instance. All profile instances and the HCL Portal product binary files are updated at the same time to avoid conflicts. If a profile belongs to a cluster and the cluster is updated, the shared HCL Portal product binary files and all other profiles must be updated to maintain synchronization.
```

Reply:
OK